content/docs/overview/about.md

```diff
@@ -16,7 +16,7 @@ seo:
 
 NuoDB Control Plane (CP) enables customers to consume NuoDB in a Database as a Service (DBaaS) model.
 It runs inside a Kubernetes cluster and handles provisioning and management of multiple domains, and their databases, in the _same_ cluster.
-NuoDB databases are created on-demand remotely using REST services by exposing various predefined configuration options.
+NuoDB databases are created on-demand remotely using REST services using various predefined configuration options.
```

Comment:
Maybe it is just me, but I still don't understand this sentence.

What are the "REST services" in question? Are you saying that the end user connects to a REST service that calls the Control Plane to create resources? That is definitely a deployment model, but it is not the only one (and not the model for the Terraform provider or external operator). Did you mean "REST clients"?

Reply:
Do you think we should not bind this sentence to a deployment model?
How about --- "NuoDB databases are created on-demand remotely by using various predefined configuration options."?

Reply:
I think my issue is that I don't understand what you mean by "REST services". Do you mean out existing clients that use the REST API? I did not think of those as "REST services", since I assume it is a service that exposes a REST API to its end users.

Reply:
Originally it was a typo and I meant `REST services` -> `REST service`. I belive it's better if we omit the way how databases are created. As you suggested we have TF SDK, external operator, `nuodb-cp`, Cockpit UI, etc.

Reply:
If you fix the typo, it makes sense. I don't have an opinion on if we should or should not mention the REST service. We do mention it later on in the document, so the reader is going to be aware of it, but as you said they may not be calling it directly.